app/modules/transactions.js

```diff
@@ -132,6 +131,7 @@ export const sendTransaction = (sendEntries: Array<SendEntryType>) => async (
       tokensBalanceMap,
       address: fromAddress,
       publicKey,
+      fees: Number(priorityFee),
```

Comment:
Use `toNumber` from `core/math.js`

Reply:
Also, where is `fees` used?

Reply:
`fees` is passed as part of the config object into `sendAsset` / `doInvoke` which will handle the rest.